atlp-rwanda/octopus-bn-frontend
# 170947380 - Users Should Receive  Reset Password Link

#### What does this PR do?
It allows a user to receive a link in their email address to reset their password.

#### Description of Task to be completed?

- send a password reset  link to the user's email address

#### How should this be manually tested?
-  Run this `git clone https://github.com/Stackup-Rwanda/octopus-bn-frontend.git` in your terminal if you don't have the project on your computer
- If you have it, run `git fetch && git checkout ft-reset-password-170947380` instead
- Run `npm run dev` command 
- In your browser go to `localhost:your_port/forgot-password`
- Enter an existing and accessible email in the field
- Go that same email address and you'll see the rese password email.

#### Screenshots (if appropriate)
 ##### Reset Password  Request Page

<img width="1440" alt="Screen Shot 2020-04-27 at 12 18 14" src="https://user-images.githubusercontent.com/25052255/80361463-3f288400-8881-11ea-9c85-1b194f5f40de.png">


#### What are the relevant pivotal tracker stories?
[#170947380](https://www.pivotaltracker.com/story/show/170947380)
